test/common/src/main/kotlin/org/codice/compliance/verification/profile/SingleSignOnProfileVerifier.kt

```diff
@@ -30,17 +30,24 @@ import org.codice.compliance.utils.TestCommon.Companion.ID
 import org.codice.compliance.utils.TestCommon.Companion.SP_ISSUER
 import org.codice.compliance.utils.TestCommon.Companion.idpMetadata
 import org.opensaml.saml.saml2.metadata.impl.EntityDescriptorImpl
+import org.slf4j.LoggerFactory
 import org.w3c.dom.Node
 
 class SingleSignOnProfileVerifier(private val response: Node, private val acsUrl: String?) {
+    companion object {
+        private val logger = LoggerFactory.getLogger(SingleSignOnProfileVerifier::class.java)
+    }
```

Comment:
❓ Is there a reason you setup a logger object here instead of using the `Log.debug` method like in the other places

Reply:
❓ How do those other logging messages print out? This one, I assume, clearly indicates it's using the `SingleSignOnProfileVerifier` logger; do the others have some generic logger source? Do we care?

Reply:
😞 My bad that's leftover code from my many attempts at using other debuggers.

Reply:
It just has one static logger. 

> It is for formatting standard output comfortably without the need to construct a Logger object. But it's also no problem to create one.